File: LeetCode/Simulation/JS/Leetcode824-山羊拉丁文.js
var toGoatLatin = function (sentence) {
    const s = sentence.split(' '), st = new Set();
    st.add('a'), st.add('e'), st.add('i'), st.add('o'), st.add('u');
    let res = '', cur = 'a';
    for (let i = 0; i < s.length; i++) {
        if (i) res += ' ';
        if (st.has(s[i][0]) || st.has(s[i][0].toLowerCase())) {
            res += s[i] + "ma";
        } else {
            res += s[i].substr(1) + s[i][0] + "ma";
        }
        res += cur;
        cur += 'a';
    }
    return res;
};

File: LeetCode/Simulation/Python/Leetcode824-山羊拉丁文.py
class Solution:
    def toGoatLatin(self, sentence: str) -> str:
        s = sentence.split(' ')
        st, cur = set("aeiou"), 'a'
        print(st)
        for i in range(len(s)):
            if s[i][0] in st or s[i][0].lower() in st:
                s[i] += 'ma'
            else:
                s[i] = s[i][1:] + s[i][0] + 'ma'
            s[i] += cur
            cur += 'a'
        return ' '.join(s)
